integration-tests/cucumber-tests-platform-common/src/test/resources/features/common/osgp-adapter-ws-admin/DeviceManagement/FilterDeviceLogItem.feature

```diff
@@ -0,0 +1,96 @@
+@Common @Platform @AdminDeviceManagement
+Feature: Filter device messages
+  As an operator
+  I want to be filter and sort the device messages 
+  In order to analyse problems and check messages
+
+  Background: 
+    Given I have 6 device log items
+      | DeviceIdentification | DEV-1                |
+      | Organization         | Liander              |
+    And I have 5 device log items
+      | DeviceIdentification | DEV-2                |
+      | Organization         | Liander              |
+    And I have 4 device log items
+      | DeviceIdentification | DEV-11               |
+      | Organization         | TEST                 |
+
+  Scenario: No filters set so getting all the messages
+    When receiving a message log request without a filter
+    Then the messages response contains 15 messages with date filter or no filter
+
+  Scenario: Use wildcard filter for Device Identification
+    When receiving a filter message log request
+      | DeviceIdentification | DEV-1* |
+    Then the messages response contains 10 messages for devices
+      | DEV-1  |
+      | DEV-11 |
+
+  @NightlyBuildOnly
```

Comment:
Strange to use this annotation in a pull request. The test isn't run until after merging. I would suggest removing it. Maybe we can add it again later.
Goes for all the usage of @NightlyBuildOnly

Reply:
Removed them.